### ISSUE TITLE
Bookmarks for multiple Chrome profiles

### ISSUE THREAD

**Issue by astanush** [blainesch/alfred-chrome-bookmarks#7]
I have two profiles set up in Chrome, Personal and Work. It seems that when I first set up this workflow, I was able to search my Work bookmarks fine. Now it seems to only search my Personal bookmarks. Note: Personal is set up as my default profile.

_Disclaimer: I am not a developer_

I looked at the script in the workflow:

```
PROFILE="~/Library/Application Support/Google/Chrome/Default/Bookmarks" php bookmarks.php {query}
```

So I dug into `~/Library/Application Support/Google/Chrome/` further, and found folders named `Profile 1` and `Profile 2`. The `Profile 1` folder does not have a `Bookmarks` JSON file, but `Profile 2` does, and the contents seem to match the bookmarks of my work profile.

Is there a way to edit the script so that it searches both JSON files?

**Comment by astanush** [comment]
Whoops, I just saw #6. Though, it was unclear if the snippet in his first comment was the solution or not.

**Comment by blainesch** [comment]
I'm not sure why #6 was closed, it looks like I never fixed it. @trentmilton did you ever update any code to work with the glob? If not I'll try and get something working this weekend.

**Comment by trentmilton** [comment]
I don't think you changed anything and neither did I.

- Regards, Trent Milton

On 18 Jul 2015, at 6:52 am, Blaine Schmeisser <notifications@github.com> wrote:

I'm not sure why #6 was closed, it looks like I never fixed it. @trentmilton did you ever update any code to work with the glob? If not I'll try and get something working this weekend.

**Comment by blainesch** [comment]
@trentmilton @astanush  fixed by 9ef9e4477f662d9d7ee54672b3a3e8e647c74b46

**Comment by astanush** [comment]
Awesome, works great! :+1:
